src/test/expression/like_matcher_test.cpp

```diff
@@ -12,7 +12,7 @@ class LikeMatcherTest : public ::testing::Test {
  public:
   bool match(const std::string& value, const std::string& pattern) const {
```

Comment:
any special reason why these are not pm_strings?

Reply:
Because it makes the test easier. Otherwise we would have to construct the pmr_string in every caller.